src/kirby/components/card/card.component.scss

```
@@ -12,7 +12,7 @@ $min-card-width: 304px;
   --kirby-card-footer-color: #{get-color('primary-contrast')};
 
   border-radius: $border-radius;
-  box-shadow: 0 10px 29px -9px rgba(24, 24, 24, 0.3);
+  box-shadow: #{get-elevation('z2')};
```

No need to wrap the function call (only needed when assigning to css variables):
```suggestion
  box-shadow: get-elevation('z2');
```